Library/.rubocop.yml

```diff
@@ -157,6 +158,7 @@ Style/DisableCopsWithinSourceCodeDirective:
   Enabled: true
   Include:
     - 'Taps/*/*/{Formula/,Casks/,}*.rb'
+    - '/**/{Formula/,Casks/,}*.rb'
```

Comment:
```suggestion
    - 'Taps/*/*/*.rb'
    - '/**/{Formula,Casks}/*.rb'
```

Reply:
@reitermarkus Why remove one directory level completely?

Reply:
What do you mean?

Reply:
A pattern of `Taps/*/*/*.rb` is no longer going to apply to `Taps/homebrew/homebrew-core/Formula/formula.rb`, or is it?

Reply:
That's already included in the second pattern.

Reply:
Ahh, now I finally get it, too 😂